modules/galera/README.md

```diff
@@ -0,0 +1,56 @@
+# MySQL Galera SignalFX detectors
+
+## How to use this module
+
+```hcl
+
+module "signalfx-detectors-database-mysql" {
+  source        = "github.com/claranet/terraform-signalfx-detectors.git//modules/database/galera?ref={revision}"
+  environment   = var.environment
+  notifications = var.notifications
+}
+```
+
+## Notes
+
+To prepare the Mysql server allowing monitoring you can use [this terraform module](https://github.com/claranet/terraform-signalfx-integrations/tree/master/database/mysql).
+
+This module is usually used in addition with the [MySQL](../database/mysql) module.
```

Comment:
will be probably "../database-mysql" when every existing modules will be moved in modules dir